index.js

```diff
@@ -118,6 +125,10 @@ async function run() {
 
 run().catch((err) => {
   console.error(err);
+  console.log(
+    "Payload that triggered this error: ",
+    JSON.stringify(context.payload)
```

Comment:
The console should be able to handle objects, so there is no need to serialize this.
```suggestion
    context.payload
```

Reply:
I'd like to have the event in JSON format on the console, as only then it can be used for debugging purposed and put in to an `event.json` file. That's why I used `JSON.stringify`.

Reply:
But the console will format the JSON by default no?

Reply:
@jonkoops - when printing on the console, it will print curly braces, but will use the wrong quotes (single quotes instead of double quotes), and will not put quotes around the keys that JSON would require. So the default will print `action: 'created'` when JSON requires `"action": "created"`.
So that's why I added an explicit call to JSON.

Reply:
Seems valid for logging, but if we could still pretty-print this with:
```suggestion
    JSON.stringify(context.payload, null, 2)
```

WDYT?